### ISSUE TITLE
[Bug]: Banner is not showing up on public pages

### ISSUE THREAD

**Issue by mberneis** [kenepa/banner#27]
### What happened?

If I have a regular public page the page (accessible without authentication) the banner does not show up.
Is there a trait I can pull in to display the banner if active?

### How to reproduce the bug

Create a custom filament page
Create a public route to this page

i.e.
```
Route::get('/app/privacy',
    Pages\Privacy::class)->withoutMiddleware([Authenticate::class])->name('filament.app.pages.privacy');
```

### Package Version

0.0.9

### PHP Version

8.2.27

### Laravel Version

11.35

### Which operating systems does with happen with?

macOS

**Comment by Jehizkia** [comment]
No, unfortunately, this is not possible yet. This is planned for future release though. Thanks!

**Comment by mberneis** [comment]
@Jehizkia - Any ETA on this?

**Comment by Jehizkia** [comment]
Hi @mberneis , No I have no ETA. But It is still planned to be implemented.
